TabloidCLI/Program.cs

```diff
@@ -1,11 +1,17 @@
 ﻿using TabloidCLI.UserInterfaceManagers;
+using System;
 
 namespace TabloidCLI
 {
     class Program
     {
         static void Main(string[] args)
         {
+
+            Console.BackgroundColor = ConsoleColor.Cyan;
+            Console.Clear();
+            Console.ForegroundColor = ConsoleColor.DarkMagenta;
```

Comment:
This is a terrifying color combination.  Well done.

Reply:
Looks good!!